enterprise/migrations/0051_add_enterprise_slug.py

```diff
@@ -0,0 +1,35 @@
+# -*- coding: utf-8 -*-
+from __future__ import unicode_literals
+
+from django.db import migrations, models
+from django.utils.text import slugify
+
+
+def populate_slug(apps, schema_editor):
+    EnterpriseCustomer = apps.get_model('enterprise', 'EnterpriseCustomer')
+
+    enterprises = EnterpriseCustomer.objects.all()
+    for enterprise in enterprises:
+        enterprise.slug = slugify(enterprise.name)
```

Comment:
Not sure we need to pre-populate these, especially if we're allowing the field to be blank

Reply:
Nevermind - I see now the subsequent migration makes this field unique, so they can't all be blank.